packages/loader/container-loader/src/collabWindowTracker.ts

```diff
@@ -28,35 +31,28 @@ import { isSystemMessage } from "@fluidframework/protocol-base";
 //    Note that system ops (including noops themselves) are excluded, so it's 1 noop per 50 real ops.
 export class CollabWindowTracker {
     private opsCountSinceNoop = 0;
-    private readonly timer: Timer;
+    private readonly timer: Timer | undefined;
 
     constructor(
         private readonly submit: (type: MessageType, contents: any) => void,
-        private readonly activeConnection: () => boolean,
-        NoopTimeFrequency: number = 2000,
-        private readonly NoopCountFrequency: number = 50,
+        NoopTimeFrequency: number = defaultNoopTimeFrequency,
+        private readonly NoopCountFrequency: number = defaultNoopCountFrequency,
     ) {
-        this.timer = new Timer(NoopTimeFrequency, () => {
-            // Can get here due to this.stopSequenceNumberUpdate() not resetting timer.
-            // Also timer callback can fire even after timer cancellation if it was queued before cancellation.
-            if (this.opsCountSinceNoop !== 0) {
-                assert(this.activeConnection(),
-                    0x241 /* "disconnect should result in stopSequenceNumberUpdate() call" */);
-                this.submitNoop(false /* immediate */);
-            }
-        });
+        if (NoopTimeFrequency !== Infinity) {
+            this.timer = new Timer(NoopTimeFrequency, () => {
+                // Can get here due to this.stopSequenceNumberUpdate() not resetting timer.
+                // Also timer callback can fire even after timer cancellation if it was queued before cancellation.
+                if (this.opsCountSinceNoop !== 0) {
+                    this.submitNoop(false /* immediate */);
+                }
+            });
+        }
     }
 
     /**
      * Schedules as ack to the server to update the reference sequence number
      */
     public scheduleSequenceNumberUpdate(message: ISequencedDocumentMessage, immediateNoOp: boolean): void {
-        // Exit early for inactive (not in quorum or not writers) clients.
```

Comment:
Is this removal intentional? Looks like we can get here with non-active connection, which may cause noops to be sent, right?